nvim/lua/plugins/lsp/servers/init.lua

```diff
@@ -96,6 +96,19 @@ M.setup = function()
 
   -- Markdown
   require("lspconfig").marksman.setup(config({}))
+
+  -- Grammar and spelling markup languages (includes markdown)
+  require("lspconfig").ltex.setup(config({
+    settings = {
+      ltex = {
+        language = "en",
+        flags = { debounce_text_changes = 300 },
```

Comment:
Flags were mistakenly nested under settings, when they should have actually be neighbours.